about.css

```diff
@@ -0,0 +1,790 @@
+  border: rgb(240, 237, 237);
+  border-width: 1px;
+  border-style: solid;
+  width: 90%;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 60px;
+  margin-bottom: 60px;
+  height: 180px;
+  position: relative;
+  background-color: #ec5242;
+  align-content: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  color: white;
+}
+
+.event img {
+  opacity: 25%;
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  left: 0%;
+}
+
+.year {
+  margin-bottom: 1px;
+  font-size: 30px;
+  font-weight: 900;
+  margin-left: 40%;
+}
+
+.e-desc {
+  margin-bottom: 1px;
+  font-size: 18px;
+  margin-left: 10%;
+  text-align: center;
+}
+
+.e-card-2 {
+  margin-top: -10%;
+}
+
+.partner {
+  background-color: #424146;
+  padding-top: 1px;
+  height: 20%;
+  margin-bottom: 0;
+}
+
+.partner .h-partner {
+  font-size: 20px;
+  color: #fff;
+  font-weight: bold;
+  text-align: center;
+  font-family: 'Lato', sans-serif;
+  margin-left: 5%;
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+.img-partner img {
+  width: 80%;
+  height: 10%;
+}
+
+.grid {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: 30% 30%;
+  height: 30%;
+  width: 100%;
+  align-items: center;
+}
+
+.main-program {
+  background-image: url(images/31.png);
+  background-repeat: inherit;
+  background-position: 100%;
+  width: 100%;
+  height: 100%;
+}
+
+.contact {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.contact-a {
+  font-size: 20px;
+  font-weight: 900;
+  font-family: 'Lato', sans-serif;
+}
+
+.m-program {
+  margin-top: -20px;
+}
+
+.m-program .h-main {
+  display: flex;
+  font-family: 'Cocogoose', sans-serif;
+  align-items: center;
+  flex-direction: column;
+  justify-content: center;
+  font-size: 18px;
+  margin-left: auto;
+  margin-right: auto;
+  text-align: center;
+  font-weight: 100;
+  margin-top: 0;
+  margin-top: 20px;
+  color: #fff;
+}
+
+.m-program .line,
+.featured-speakers .line {
+  width: 20%;
+  border-bottom: 2px solid #ec5242;
+}
+
+.card {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  width: 82%;
+  height: 70%;
+  margin-left: auto;
+  margin-right: auto;
+  background-color: rgb(255, 255, 255, 0.1);
+  border: 2px solid transparent;
+  font-family: 'Cocogoose', sans-serif;
+}
+
+.card .fas {
+  padding: 30px;
+  margin: auto;
+  font-size: 40px;
+}
+
+.item {
+  width: 20%;
+  height: 15%;
+}
+
+.a {
+  margin-right: 40%;
+}
+
+.b {
+  margin-left: -30%;
+  color: #ec5242;
+}
+
+.c {
+  margin-left: 10%;
+  color: white;
+  width: 100%;
+}
+
+.featured-speakers .h-featured {
+  display: flex;
+  font-family: 'Cocogoose', sans-serif;
+  align-items: center;
+  flex-direction: column;
+  justify-content: center;
+  font-size: 18px;
+  margin-left: auto;
+  margin-right: auto;
+  text-align: center;
+  font-weight: 100;
+  margin-top: 0;
+  margin-top: 20px;
+  color: #000;
+}
```

Comment:
- [ ] We also need to make a little adjustment to the featured speakers' section.

| **The Provided Design** |
| --- |
| ![featured-design](https://user-images.githubusercontent.com/45002654/133836028-f6e4d9d3-b6e7-4b4a-aafa-331b8cb6a14c.PNG) |

| **Your Implementation** |
| --- |
| ![featured-imp](https://user-images.githubusercontent.com/45002654/133836083-6815f302-e013-40c8-8db6-1ea43f20b942.PNG) |

- The images should have the same height and width as in the original design. Your images need some width, they are narrow
- The image applied in the background can be made smaller so that focus can remain in the image in front.
- Margin bottom can really help visually separate each speaker section from the one below it.
- Typography is not maintained, let's not ignore font style, i.e. italics. The text below the speaker's name should be italicized
- Try and match your design with the provided one as much as possible.